rest-catalog/rest-catalog-service/src/main/java/org/projectnessie/restcatalog/service/resources/IcebergV1ConfigResource.java

```diff
@@ -30,6 +40,28 @@ public ConfigResponse getConfig(String warehouse) {
     config.withDefaults(w.configDefaults());
     config.withOverrides(w.configOverrides());
 
+    // The following properties are passed back to clients to automatically configure their Nessie
+    // client. These properties are _not_ user configurable properties.
+    config.withOverride("nessie.default-branch.name", tenantSpecific.defaultBranch().name());
+    config.withOverride("nessie.is-nessie-catalog", "true");
+    // Make sure that `nessie.core-base-uri` always returns a `/` terminated URI.
+    config.withOverride("nessie.core-base-uri", uriInfo.getBaseUri().resolve("api/").toString());
```

Comment:
One last nit here: why not use `CONF_NESSIE_URI` for this? I don't see any value in creating a separate setting.